tmt/utils/__init__.py

```diff
@@ -5310,6 +5310,29 @@ def _process_step_collection(step_name: str, step_collection: Any) -> None:
     return node
 
 
+def preformat_jsonschema_validation_errors(
```

Comment:
@happz this function would deserve a unit test to have some expectation how the formatting works, in case changs would be made

Reply:
Added a better docstring in ac31e8e9. Hopefully, this function will go away Soon(TM), and make space for better error reporting, therefore I'd prefer not to spend too much time with it.